inference-engine/tests/ie_test_utils/functional_test_utils/test_model/exec_graph_serialization_model.hpp

```diff
@@ -0,0 +1,281 @@
+// Copyright (C) 2019 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <string>
+
+namespace FuncTestUtils {
+
+const char serialize_test_model[] = R"V0G0N(<?xml version="1.0" ?>
```

Comment:
I am not sure that we need to have these models in the header file.

Can we move these models to the test cpp file?